sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ui/HiveThriftServer2ListenerSuite.scala

```diff
@@ -19,8 +19,8 @@ package org.apache.spark.sql.hive.thriftserver.ui
 
 import java.util.Properties
 
-import org.mockito.Mockito.{mock, RETURNS_SMART_NULLS}
 import org.scalatest.BeforeAndAfter
+import org.scalatestplus.mockito.MockitoSugar.mock
```

Comment:
I don't think it matters.

Reply:
Ok, then I will close this PR in favour of #26707.